src/app/exercise/[equipment_type]/[exercise_type]/edit/[exercise_id]/_components/EditBarbellExercise/index.tsx

```diff
@@ -101,7 +94,7 @@ const useEditBarbellExerciseAPI = (props: EditBarbellExerciseProps) => {
 
   return {
     targetWeight: localTargetWeight,
-    onTargetWeightWeightChange: onLocalTargetWeightChange,
+    onTargetWeightWeightChange: setLocalTargetWeight,
```

Comment:
Another instance of change needed since I changed the type to be the set state action